model/error/_meta/fields.yml

```diff
@@ -357,7 +357,13 @@
         - name: connection_type
           type: keyword
           description: >
-            Cellular network technology, eg. 4G
+            Network type, eg. "wifi", "cell"
+          overwrite: true
+
+        - name: connection_subtype
```

Comment:
Should we change to `connection.type` and `connection.subtype` on the document level?

Reply:
I haven't made the `.type` & `.subtype` updates to the agent yet, since the spec hasn't been merged yet. So the iOS agent is still sending `net.host.connection_type` and not yet sending anything related to `subtype`.

Reply:
In this case I suggest we keep what we have for `7.14` and keep this PR open until the spec is merged.

Reply:
I've made this change now.